lib/web_console/railtie.rb

```diff
@@ -55,9 +55,9 @@ class Railtie < ::Rails::Railtie
       end
     end
 
-    initializer 'web_console.whiny_requests' do
-      if config.web_console.key?(:whiny_requests)
-        Middleware.whiny_requests = config.web_console.whiny_requests
+    initializer 'web_console.verbose_logging' do
+      if config.web_console.key?(:verbose_logging)
```

Comment:
We can't just rename that, its already documented [here](https://github.com/rails/web-console#configweb_consolewhiny_requests). Even if its not _just_ for requests anymore, we don't wanna break any applications using the previous option.